lms/static/sass/views/_verification.scss

```diff
@@ -244,6 +244,42 @@
       }
     }
   }
+  .report-receipt-provider {
+        @extend %ui-window;
+        border-top:3px solid $credit-color-base !important;
+        p {
+          padding: 0px $baseline $baseline/2 $baseline;
+          overflow: auto;
+        }
+        .bold_param {
+          padding: $baseline*.75 $baseline 0px $baseline;
+          font-weight: 600;
+        }
+        .bold_param span {
+          float: right;
+        }
+        div {
+          padding: 10px 20px;
+          margin: 0 0 15px;
+          overflow: auto;
+        }
+        div span {
+          float: right;
+          padding: 0;
+          margin: 0;
+        }
+        .complete-course {
+          @extend %btn-pl-primary-base;
+          @include float(right);
+
+          &.archived {
+            @extend %btn-pl-default-base;
+          }
+        }
+        .custom_instructions div {
+          float: left;
+        }
+      }
```

Comment:
@AlasdairSwan Could I trouble you for a quick review of these styles?  This is for the credit provider details on the receipt page:

https://openedx.atlassian.net/wiki/pages/viewpage.action?pageId=30967130